packages/notifi-react-card/lib/components/subscription/subscription-card-views/preview-panel/UserInfoPanel.tsx

```diff
@@ -17,20 +17,46 @@ export type UserInfoPanelProps = {
     telegram?: UserInfoSection;
     sms?: UserInfoSection;
     EditButton: string;
+    confirmation: {
+      email: string;
+      emailLabel: string;
+      telegram: string;
+      telegramLabel: string;
+    };
   }>;
```

Comment:
we can probably move these into the `UserInfoSection` as confirmationLink and confirmationLabel